flowkit_ui_backend/impl/data_api_impl.py

```diff
@@ -297,13 +309,15 @@ async def run_query(
         table_name=table_name,
     )
     # get temporal resolution to know how to format the spatial entities
-    tr = await get_temporal_resolution(query_parameters.trid, token_model=token_model, pool=pool)
+    tr = await get_temporal_resolution(
+        query_parameters.trid, token_model=token_model, pool=pool
+    )
 
     metadata = await get_metadata(query_parameters, tr, pool, token_model)
 
     # support getting mdids only
     if query_parameters.mdids_only is True:
-        return QueryResult(mdids=[m.mdid for m in metadata])
+        return QueryResult(mdids=[f"{m.mdid}" for m in metadata])
```

Comment:
Does `m.mdid` kick out an int here by default?

Reply:
Yeah, it is an int in the db so gets typed